src/main/kotlin/no/nav/familie/tilbake/micrometer/domain/StatistikkRepository.kt

```diff
@@ -0,0 +1,145 @@
+              FROM fagsak
+              JOIN behandling ON fagsak.id = behandling.fagsak_id
+              JOIN behandlingsresultat ON behandling.id = behandlingsresultat.behandling_id
+              AND behandlingsresultat.type LIKE 'HENLAGT%'
+              JOIN behandlingsvedtak ON behandlingsresultat.id = behandlingsvedtak.behandlingsresultat_id
+              AND behandlingsvedtak.iverksettingsstatus = 'IVERKSATT'
+              WHERE status = 'AVSLUTTET'
+              GROUP BY ytelsestype, vedtaksdato""")
+    fun finnVedtakHenlagte(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM okonomi_xml_mottatt_arkiv
+              GROUP BY ytelsestype, dato""")
+    fun finnKobledeKravgrunnlag(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM okonomi_xml_mottatt
+              GROUP BY ytelsestype, dato""")
+    fun finnUkobledeKravgrunnlag(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, b.opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM fagsak
+              JOIN behandling ON fagsak.id = behandling.fagsak_id
+              JOIN brevsporing b ON behandling.id = b.behandling_id
+              AND b.brevtype = 'VARSEL'
+              GROUP BY ytelsestype, dato""")
+    fun finnSendteVarselbrev(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, b.opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM fagsak
+              JOIN behandling ON fagsak.id = behandling.fagsak_id
+              JOIN brevsporing b ON behandling.id = b.behandling_id
+              AND b.brevtype = 'KORRIGERT_VARSEL'
+              GROUP BY ytelsestype, dato""")
+    fun finnSendteKorrigerteVarselbrev(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, b.opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM fagsak
+              JOIN behandling ON fagsak.id = behandling.fagsak_id
+              JOIN brevsporing b ON behandling.id = b.behandling_id
+              AND b.brevtype = 'VEDTAK'
+              GROUP BY ytelsestype, dato""")
+    fun finnSendteVedtaksbrev(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, b.opprettet_tid::DATE AS dato, COUNT(*) AS antall
+              FROM fagsak
+              JOIN behandling ON fagsak.id = behandling.fagsak_id
+              JOIN brevsporing b ON behandling.id = b.behandling_id
+              AND b.brevtype = 'HENLEGGELSE'
+              GROUP BY ytelsestype, dato""")
+    fun finnSendteHenleggelsesbrev(): List<ForekomsterPerDato>
+
+    // language=PostgreSQL
+    @Query("""SELECT ytelsestype, b.opprettet_tid::DATE AS dato, COUNT(*) AS antall
```

Comment:
Kan vel være en ide og begrense hvor langt tilbake i tid vi henter data for i neste PR?

Reply:
Ja. Endringer jeg ser for meg er reduksjon i frekvens på spørringene, tidsavgrensning bakover og mulig behov for indekser for å lette spørringene.